### ISSUE TITLE
defining changelogPresets throws an error

### ISSUE THREAD

**Issue by ayZagen** [ghiscoding/lerna-lite#728]
### Describe the bug

when trying to run `pnpm lerna version --dry-run` it throws following error if `lerna.json` config includes `changelogPresets`

```txt
/node_modules/.pnpm/npm-package-arg@11.0.1/node_modules/npm-package-arg/lib/npa.js:33
  const nameEndsAt = arg[0] === '@' ? arg.slice(1).indexOf('@') + 1 : arg.indexOf('@')
                        ^

TypeError: Cannot read properties of undefined (reading '0')
    at npa (/node_modules/.pnpm/npm-package-arg@11.0.1/node_modules/npm-package-arg/lib/npa.js:33:25)

```

### Reproduction

using posted lerna.json run `pnpm lerna version --dry-run`

### Lerna config and logs

<details><summary>lerna.json</summary><p>

```json
{
  "$schema": "node_modules/@lerna-lite/cli/schemas/lerna-schema.json",
  "version": "independent",
  "independent": true,
  "changelog": true,
  "conventionalCommits": true,
  "changelogPreset": {
    "commitUrlFormat": "{{host}}/{{owner}}/{{repository}}/commit/{{hash}}",
    "compareUrlFormat": "{{host}}/{{owner}}/{{repository}}/compare/{{previousTag}}...{{currentTag}}",
    "issueUrlFormat": "{{host}}/{{owner}}/{{repository}}/issues/{{id}}",
    "userUrlFormat": "{{host}}/{{user}}"
  },
  "packages": [
    "packages/*"
  ]
}
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
0 silly argv [object Object]
1 notice cli v2.5.1
2 verbose rootPath <REDACTED>
3 info [1m[35m[dry-run] >[39m[22m git rev-parse
4 info versioning independent
5 verbose project packages services/* dashboard models libs/saml
6 silly isAnythingCommitted
7 info [1m[35m[dry-run] >[39m[22m git rev-list --count --all --max-count=1
8 verbose isAnythingCommitted 1
9 silly currentBranch
10 verbose currentBranch main
11 silly remoteBranchExists
12 info [1m[35m[dry-run] >[39m[22m git show-ref --verify refs/remotes/origin/main
13 silly isBehindUpstream
14 info [1m[35m[dry-run] >[39m[22m git remote update
15 info [1m[35m[dry-run] >[39m[22m git rev-list --left-right --count origin/main...main
16 silly isBehindUpstream main is behind origin/main by NaN commit(s) and ahead by undefined
17 silly hasTags
18 verbose hasTags filter for tags with '*@*' pattern
19 verbose hasTags false
20 info Assuming all packages changed
21 verbose updated @packages/core
22 verbose updated @packages/backend
23 verbose updated @packages/frontend
24 verbose updated @packages/tsconfig
25 verbose updated @packages/config
26 verbose updated @packages/test
27 info [1m[35m[dry-run] >[39m[22m git describe --always --long --dirty --first-parent
28 silly independent for @pa/dashboard at <REDACTED>/dashboard
29 error TypeError: Cannot read properties of undefined (reading '0')
29 error     at npa (<REDACTED>/node_modules/.pnpm/npm-package-arg@11.0.1/node_modules/npm-package-arg/lib/npa.js:33:25)
29 error     at GetChangelogConfig.getChangelogConfig (file://<REDACTED>/node_modules/.pnpm/@lerna-lite+version@2.5.1_@lerna-lite+publish@2.5.1_typescript@5.2.2/node_modules/@lerna-lite/version/dist/conventional-commits/get-changelog-config.js:33:28)
29 error     at recommendVersion (file://<REDACTED>/node_modules/.pnpm/@lerna-lite+version@2.5.1_@lerna-lite+publish@2.5.1_typescript@5.2.2/node_modules/@lerna-lite/version/dist/conventional-commits/recommend-version.js:31:48)
29 error     at file://<REDACTED>/node_modules/.pnpm/@lerna-lite+version@2.5.1_@lerna-lite+publish@2.5.1_typescript@5.2.2/node_modules/@lerna-lite/version/dist/version-command.js:346:62
29 error     at iterator (file://<REDACTED>/node_modules/.pnpm/@lerna-lite+version@2.5.1_@lerna-lite+publish@2.5.1_typescript@5.2.2/node_modules/@lerna-lite/version/dist/version-command.js:278:64)

```
</p></details>

### Environment Info

  System:
    OS: Linux 6.5 openSUSE Tumbleweed 20230917
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Memory: 23.11 GB / 31.29 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh
  Binaries:
    Node: 20.6.1 - ~/.nvm/versions/node/v20.6.1/bin/node
    npm: 9.8.1 - ~/.nvm/versions/node/v20.6.1/bin/npm
    pnpm: 8.7.6 - ~/.nvm/versions/node/v20.6.1/bin/pnpm
  Browsers:
    Chrome: 118.0.5993.11
  npmPackages:
    @lerna-lite/changed: ^2.5.1 => 2.5.1 
    @lerna-lite/cli: ^2.5.1 => 2.5.1 
    @lerna-lite/publish: ^2.5.1 => 2.5.1 
    @lerna-lite/version: ^2.5.1 => 2.5.1 

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/lerna-lite/lerna-lite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/lerna-lite/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/lerna-lite/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/lerna-lite/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
latest v2.5.1 had an upgrade of `conventional-commits-parser` to a major version in PR #709 

Can you try to use fixed version `2.5.0` to see if you still have that problem? If that is the case, we could revert #709

**Comment by ghiscoding** [comment]
it's kinda hard to help without a full repro, does it work properly if you remove the `changelogPreset`?
